lib/sqlalchemy/dialects/postgresql/base.py

```diff
@@ -2279,6 +2280,12 @@ def visit_create_index(self, create, **kw):
                 [preparer.quote(c.name) for c in inclusions]
             )
 
+        nulls_not_distinct = index.dialect_options["postgresql"]["nulls_not_distinct"]
```

Comment:
the default is `None` that means nothing is rendered, correct?

Reply:
Yes. Will be rendered only if value is  `True` or `False`.
True - `NULLS NOT DISTINCT`
False - `NULLS DISTINCT`